components/regionDropdowns/BaseRegionsDropdownItem.tsx

```diff
@@ -0,0 +1,32 @@
+import { Menu } from "@headlessui/react";
+import { CheckIcon } from "@heroicons/react/outline";
+import clsx from "clsx";
+import React from "react";
+
+interface BaseRegionsDropdownItemProps {
+  label: string;
+  chosen: boolean;
+  onClick: () => void;
+}
+
+export function BaseRegionsDropdownItem({ label, chosen, onClick }: BaseRegionsDropdownItemProps) {
```

Comment:
I wonder if this is in any way a `regions` dropdown item right now? Looking at the code it seems like a regular item. Is the styling regions specific? Maybe then it'd be worth to just have a dropdown item with styles passed  to it?

Reply:
I'll move to the generic dropdown when we integrate SRS with the store UI. It will be easier to me to generalize dropdown component.

Reply:
Okay